testing_and_setup/compass/ocean/global_ocean/scripts/create_E3SM_coupling_files.ini

```diff
@@ -0,0 +1,27 @@
+[main]
+mesh_name = oQU240
+
+[initial_condition_ocean]
+enable = true
+
+[graph_partition_ocean]
+enable = true
+
+[initial_condition_seaice]
+enable = true
+
+[scrip]
+enable = true
+
+[transects_and_regions]
+enable = true
+transect_region_geojson = /usr/projects/climate/mpeterse/analysis_input_files/geojson_files/SingleRegionAtlanticWTransportTransects.geojson
+
+[mapping_Gcase]
+enable = true
+atm_scrip_path = /lustre/scratch3/turquoise/mpeterse/E3SM/input_data/share/scripgrids
+
+[domain]
+enable = true 
+# compiled executable gen_domain, code in E3SM repo:
+gen_domain = /usr/projects/climate/mpeterse/repos/E3SM/compiled_cime_tools/cime/tools/mapping/gen_domain_files/src/gen_domain
```

Comment:
I'm really concerned that these hard coded paths defeat the purpose of having COMPASS try to be a self-contained, freely available framework with as few preparatory steps as possible.  These things can *only* be done on LANL IC by someone in the `climate` group.

I'll let this go through as a **temporary** solution as long as it's accompanied by an issue tracking the need to generalize this ASAP.

Reply:
I completely agree. It is on my to do list to move everything out to geometric features, e3sm_unified, or the E3SM inputdata repo. But at least we have a list of what is left outside the repos right now.

Reply:
@xylar could we add gen_domain to e3sm_unified? I'm not sure if that means add it to MPAS-Tools, or somewhere else. It is this code:
https://github.com/E3SM-Project/E3SM/tree/master/cime/tools/mapping/gen_domain_files